grammars/julia.cson

```diff
@@ -218,11 +218,15 @@ repository:
         name: "keyword.operator.arrow.julia"
       }
       {
-        match: "(?:\\.==|!=|\\.>=|\\.>=|\\.>|\\.<|==|\\.!=|\\.=|\\.!|<:|:>|>=|<=|>|<)"
+        match: "(?:<<(?!=)|>>>(?!=)|>>(?!=|>=)|\\.>>>|\\.>>|\\.<<)"
+        name: "keyword.operator.shift.julia"
+      }
+      {
+        match: "(?:===|\\.==|!==|!=|\\.>=|\\.>(?!=)|\\.<(?!=)|==|\\.!=|\\.=|\\.!|<:|:>|(?<!>)>=|(?<!<)<=|\\.<=|>(?!>)|<(?!<))"
         name: "keyword.operator.relation.julia"
       }
       {
-        match: "(?::=|\\+=|-=|\\*=|//=|/=|\\.//=|\\.\\*=|\\\\=|\\.\\\\=|^=|\\.^=|%=|\\|=|&=|\\$=|<<=|>>=|=)"
+        match: "(?::=|\\+=|-=|\\*=|//=|/=|\\.//=|\\./=|\\.\\*=|\\\\=|\\.\\\\=|\\^=|\\.\\^=|%=|\\.%=|÷=|\\.÷=|\\|=|&=|\\$=|<<=|>>=|>>>=|=)"
```

Comment:
Can't these be moved above `keyword.operator.shift` to get rid of those negative lookaheads?